splice_spark/src/main/spark2.1/com/splicemachine/spark/splicemachine/SplicemachineContext.scala

```diff
@@ -39,10 +39,16 @@ import org.apache.hadoop.hbase.util.Bytes
 import org.apache.log4j.Logger
 import org.apache.spark.api.java.JavaRDD
 import org.apache.spark.scheduler.{SparkListener, SparkListenerApplicationEnd}
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings
 
+@SerialVersionUID(20200513211L)
 object Holder extends Serializable {
   @transient lazy val log = Logger.getLogger(getClass.getName)
 }
+@SerialVersionUID(20200513212L)
+@SuppressFBWarnings(value = Array("ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD"), justification = "Need to set SpliceClient.connectionString")
+@SuppressFBWarnings(value = Array("NP_ALWAYS_NULL"), justification = "These fields usually are not null")
```

Comment:
What fields was it complaining about?

Reply:
Wrote https://splicemachine.atlassian.net/browse/DB-9580 and listed the issues in the jira.